modules/savedb.py

```diff
@@ -0,0 +1,60 @@
+import os
+import socket
+import pymongo
+
+original_socket = socket.socket
+
+url = os.getenv("DATABASE_URL")
+passwd = os.getenv("DATABASE_PASSWORD")
+user = os.getenv("DATABASE_USERNAME")
+database = os.getenv("DATABASE_NAME")
+
+def check_db_options():
+    """ Just to check whether database
+    parameters are loaded
+    """
+    if not (url and passwd and user and database):
+        print("Could not load Database Configurations")
+        return 1
+
+def saveToDatabase(links):
+    """
+    Connects to a MongoDB
+    Create Mongo Collection
+    Add the links to the collection
+    Args:
+        url = url of the mongo server
```

Comment:
Could you change this and the others to ```arg_name (type): what it represents```? We use [Google's Python](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) documentation format if this helps.